### PR TITLE
First pass on DELETE forms and requests.

### PR DESCRIPTION
**Pull request by changemewtf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,175 @@
...
+  <input name="utf8" type="hidden" value="&#x2713;" />
+  <input name="authenticity_token" type="hidden" value="f755bb0ed134b76c432144748a6d4b7a7ddf2b71" />
+  <input name="commit" type="submit" value="Delete Caligula" />
+</form>
+```
+
+Enhance!
+
+> <form accept-charset="UTF-8" action="/people/1" method="**post**">
+>   <input name="_method" type="hidden" value="**delete**" />
```
Comment (ahimmelstoss) [inline README.md:69]: I like this exposition a lot. I like how you show the HTML and build up to this.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,175 @@
...
+an admin control panel where users can be deleted from a list?
+
+```erb
+<!-- app/views/people/index.html.erb //-->
+
+<% @people.each do |person| %>
+<div class="person">
+  <span><%= person.name %></span>
+  |
+  <%= link_to "Delete", person, method: :delete, data: { confirm: "Really?" } %>
```
Comment (ahimmelstoss) [inline README.md:157]: I think it would be good to include the `link_to` HTML that this generates.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,175 @@
...
+
+[url_helper_link_to]: http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to
+
+> This modifier will dynamically create an HTML form and immediately submit the
+> form for processing using the HTTP verb specified.
+
+This uses JavaScript, and it will use `GET` (the default method used by all
+browsers for HTML links) if the user has JavaScript disabled.
+
+This example also uses the `:data` option to make sure the user is ready to
```
Comment (ahimmelstoss) [inline README.md]: What happens when you include this data option? I'd mention how it pops up a confirmation window.

File: README.md (hunk 1 above)
Comment (jmburges) [inline README.md:69]: agreed!

**Pull request merged**